docs/pages/2019_MARVEL_Psik_MaX/sections/calculations.rst

```diff
@@ -3,30 +3,33 @@
 Submit, monitor and debug calculations
 ======================================
 
-The goal of this section is to understand how to create new data in
-AiiDA. We will launch a total energy calculation and check its results.
-We will introduce intentionally some common mistakes along the process
-of defining and submitting a calculation and we will explain you how to
-recognize and correct them. While this debugging is done here
-‘manually’, workflows (that we will learn later in this tutorial) can
-automate this procedure considerably. For computing the DFT energy of
-the silicon crystal (with a PBE functional) we will use Quantum
-ESPRESSO , in particular the PWscf code (``pw.x``). Besides the
-AiiDA-core package, a number of plugins exist for many different codes.
-These are listed in the `AiiDA plugin registry <https://aiidateam.github.io/aiida-registry/>`_.
-In particular, the ``aiida-quantumespresso`` plugin (already installed in
-your machine) provides a very extensive set of plugins, covering most
-(if not all) the functionalities of the underlying codes.
+In this section we'll be learning how to create new data in AiiDA. 
+
+We will use the `Quantum Espresso <https://www.quantum-espresso.org/>`_ package to
+launch a simple `density functional theory
+<https://en.wikipedia.org/wiki/Density_functional_theory>`_ calculation of 
+the silicon crystal using the  :doi:`PBE exchange-correlation functional <10.1103/PhysRevB.54.16533>`
+and check its results.  
+While we're going to debug these issues ‘manually’ here, workflows (which you'll 
+encounter later in this tutorial) can help you avoid these issues systematically.
+
+Note that besides the ``aiida-quantumespresso`` plugin, AiiDA comes with
+plugins for a range of other codes,
+all of which are listed in the `AiiDA plugin registry <https://aiidateam.github.io/aiida-registry/>`_.
 
 The AiiDA daemon
 ----------------
 
 First of all, check that the AiiDA daemon is actually running. The AiiDA
-daemon is a program running all the time in the background, checking if
-new calculations appear and need to be submitted to the scheduler. The
-daemon also takes care of all the necessary operations before the
-calculation submission, and after the calculation has completed on the
-cluster. Type in the terminal
+daemon is a program that
+
+ * runs continuously in the background
```

Comment:
This seems to suggest that it only does calculations, while it also does work chains of course. I understand we might not want to give much info here, but it might become confusing later on, if it is never explained

Reply:
Noted. I think we can explain this where it becomes relevant.